### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by faloi**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @AngeliM96

### CODE REVIEW

**Comment by AngeliM96** [comment]
@faloi Ya comenté todos los tipos, agregué los diagramas de clase y formatié, después de un par de intentos, el código markdown.

File: README.md [hunk 1]
```diff
@@ -16,6 +16,22 @@ Elaborar e incluir aquí diagramas de clase para esta implementación, con dos v
 * un solo diagrama con todas las clases,
 * tres diagramas distintos: uno para las plagas, otro para los elementos, otro para el barrio.
 
+# Diagramas de clase
+Comentario: No lo agregué en los diagramas, pero elemento creo que debería ser un tipo nuevo 'Elemento', esto también está comentado en los distintos archivos pero me pareció relevante mencionarlo.
```
Comment (faloi) [inline README.md:20]: Buena observación. En Kotlin inevitablemente va a haber que definir ese tipo para que el código funcione.

File: src/barrio.wlk [hunk 2]
```diff
@@ -1,22 +1,30 @@
-class Barrio {
+class Barrio 
+{
 
-	const property elementos = []
+	const property elementos = [] // tipo lista
```
Comment (faloi) [inline src/barrio.wlk:4]: ¿Lista de qué?

File: src/barrio.wlk [hunk 3]
```diff
@@ -1,22 +1,30 @@
...
+								   // elemento sería un tipo nuevo 'Elemento'
+	{
+		elementos.remove(elemento) // elemento sería un tipo nuevo 'Elemento'
 	}
 
-	method elementosBuenos() {
-		return elementos.count{ e => e.esBueno() }
+	method elementosBuenos() // retorna un int
+	{
+		return elementos.count{ e => e.esBueno() } // de nuevo, oxidado con wollok, pero creo que .count era solo un contador de la cantidad de
```
Comment (faloi) [inline src/barrio.wlk:20]: Exactamente eso.

File: src/elementos.wlk [hunk 4]
```diff
@@ -1,52 +1,62 @@
...
+	method fueAtacado(plaga) // no retorna nada, tipo void, es una acción
+	{
 		nivelMugre += plaga.nivelDanio()
 	}
 
 }
 
-class Mascota {
+class Mascota 
+{
```
Comment (faloi) [inline src/elementos.wlk:19]: Por favor, decime que no vas a usar este estilo de indentación de acá en adelante... :cry:

**Pull request closed without merging**